samples/kaspresso-sample/src/androidTest/kotlin/com/kaspersky/kaspressample/device_tests/DeviceLogcatSampleTest.kt

```diff
@@ -119,11 +126,11 @@ class DeviceLogcatSampleTest : TestCase() {
             step("Using reader block") {
                 repeat(100) { testLogger.i("Test6Row$it") }
 
-                var fullLogcatList = device.logcat.readLogcatRows()
+                val fullLogcatList = device.logcat.readLogcatRows()
 
-                var inneContainsSize = 0
-                val isContainsBreaked = device.logcat.readLogcatRows { logcatRow ->
-                    inneContainsSize++
+                var innerContainsSize = 0
+                val doesContainBreaked = device.logcat.readLogcatRows { logcatRow ->
```

Comment:
`is` instead of `does`?

Reply:
Why? "Does" is a modal verb as an "is" and fits English rules